src/views/Clients/ViewClients/index.jsx

```diff
@@ -78,21 +83,19 @@ export default class ViewWorker extends PureComponent {
   };
 
   handleClientSearchSubmit = e => {
-    e.preventDefault();
+    if (e) {
+      e.preventDefault();
+    }
```

Comment:
Since we won't be calling this method from `componentDidMount` anymore, we can remove the if statement and just use `e.preventDefault()`.